Cabal-syntax/src/Distribution/Types/LibraryVisibility.hs

```diff
@@ -20,7 +20,7 @@ import qualified Text.PrettyPrint                as Disp
 -- @since 3.0.0.0
 --
 data LibraryVisibility
-      -- | Can be depenendent from other packages
+      -- | Can be independent from other packages
```

Comment:
Can be depended on by other packages (is this correct english?)

Reply:
since 3.0.0.0 

presumably the code is 'trustworthy"

so if the code can be relied on/upon by other packages

possibly

`      -- | Can be relied on by other packages`

Reply:
do you mean

`      -- | API exported for use by other packages`

Reply:
`Can be used as a dependency for other packages` is closest imo. The original fix is actually quite wrong. Were you just running a spell checker + replacement recursively on the project?

Reply:
the Member comment was committed